includes/definitions/radlan.yaml

```diff
@@ -8,3 +8,5 @@ over:
 discovery:
     - sysDescr:
         - AT-8000
+        - '48-port 10/100/1000 Ethernet Switch'
```

Comment:
This seems too generic to me. How come this is needed?

Reply:
Yeah, it's a bit rubbish.  The system description is literally "48-port 10/100/1000 Ethernet Switch". Doesn't say vendor or model etc. 

Was this the correct way of specifying multiple SysDescr to potentially match on? I was trying to do "AT-8000" OR "48-port 10/100/1000 Ethernet Switch".

Reply:
Better to use sysObjectID if it has a unique one?

Reply:
Fair point. 

I'll just remove the line from this commit anyway.

As above, how do you go between "This AND that", and "This OR that" in Yaml ?

Reply:
You have OR right now with these two lines:

```
        - AT-8000
        - '48-port 10/100/1000 Ethernet Switch'
```

Basically meaning either.

For and:

```
    - sysObjectId:
        - .1.3.6.1.4.1.1129.1.2.
    - sysDescr:
        - TOSHIBA e-STUDIO
```

This requires both of those to match.